tests/torch/fx/test_models.py

```diff
@@ -28,18 +28,28 @@
 from torch._export import capture_pre_autograd_graph
 
 import nncf
+from nncf.common.graph.graph import NNCFGraph
 from nncf.common.graph.graph import NNCFNodeName
 from nncf.common.graph.operator_metatypes import OperatorMetatype
 from nncf.common.utils.os import safe_open
 from nncf.experimental.torch.fx.nncf_graph_builder import GraphConverter
 from nncf.quantization.advanced_parameters import AdvancedQuantizationParameters
 from nncf.torch.dynamic_graph.patch_pytorch import disable_patching
+from tests.cross_fw.shared.nx_graph import compare_nx_graph_with_reference
 from tests.cross_fw.shared.paths import TEST_ROOT
 from tests.torch import test_models
-from tests.torch.test_compressed_graph import check_graph
 
-FX_DIR_NAME = Path("fx")
-FX_QUANTIZED_DIR_NAME = Path("fx") / "quantized"
+
+def check_fx_graphs(graph: NNCFGraph, path_to_dot: str, graph_dir: str):
```

Comment:
Is not fx specific change, please add new arg `extended` to existed function

Reply:
All references graphs will be placed back to directory "tests/torch/data/reference_graphs/fx" and other references like metatypes will be placed to "tests/torch/data/fx/reference_metatypes", you ok with that?

Reply:
This behavior changes in this PR? Please write about all changes in PR description.

But i suggest firstly discuss about changes in structure of tests on meeting.

Why do you prefer `tests/torch/data/fx` but not `tests/torch/fx/data`?

Reply:
I have no preferences, please tell me the right structure and I'll do it

Reply:
If you have no preferences about it, lets keep structure as it.
And will discuss about structure on meeting.